src/viewport.cpp

```diff
@@ -1305,7 +1305,7 @@ void Viewport::OnMouseMoveEvent(const Point16 &pos)
 
 WmMouseEvent Viewport::OnMouseButtonEvent(MouseButtons state, WmMouseEventMode mode)
 {
-	if (mode != WMEM_PRESS) return WMME_NONE;  // Repeated buttons should be ignored for all actions below.
+	if (mode != WMEM_PRESS) return WMME_NONE;  // Avoid processing multiple events for the same click for all actions below.
```

Comment:
This was my idea due to not knowing what a repeated button was, you may want to rephrase it?